chart/values.yaml

```diff
@@ -0,0 +1,151 @@
+
+# When set to true use the `run.yaml` file in the `files/run.yaml` directory
+customRunConfig: false
+
+# TODO: Currently we are only working for vLLM this should be expanded in the future
+vllm:
+  url: "http://vllm-server"
+  inferenceModel: "llama2-7b-chat"
+  # This is the API key for the VLLM server. It can be set in two ways through a secret:
+  # TODO: Implement this
+  # secret:
+  #   name: vllm-secret
+  #   key: vll
+  # or directly with an api key (should be avoided in production)
+  # apiKey: "xxxxxxxxxxxx"
+
+# https://llama-stack.readthedocs.io/en/latest/distributions/selection.html
+distribution: distribution-remote-vllm
+
+telemetry:
+  enabled: false
+  serviceName: "otel-collector.openshift-opentelemetry-operator.svc.cluster.local:4318"
+  sinks: "console,sqlite,otel"
+
+#  Use to allow for other env variables to be passed to the container
+# env:
+#   MY_CUSTOM_ENV_VAR: "my-custom-env-var-value"
+
+replicaCount: 1
+
+# This sets the container image more information can be found here: https://kubernetes.io/docs/concepts/containers/images/
+image:
+  repository: docker.io/llamastack/{{ $.Values.distribution }}
```

Comment:
I'm not a huge fan of self referential values files.  It isn't a commonly used feature and can be a bit confusing for someone attempting to consume the chart.

I would rather see this logic moved into the chart itself, or just duplicate the value personally.

Reply:
I think I am going to push back on this one. I understand the concern but feel like this is the cleanest solution to what I want to do. 

Which is to have basically two flows.

1. The person who does not modify the chart and just changes the required values (i.e. the distribution) to suite their needs. This would be for initial use and demos

2. The person who knows what they are doing and just comes in and completely overrides the image. I also feel like the person that is used to helm charts created this way would understand the value can be over written.

That being said if other folks agree with Trevor let me know fine with changing it if multiple people think this way.

Reply:
That makes sense.

I don't have an immediately apparent cleaner way to do accomplish this other than hard coding and I could see that `distribution` value becoming very important in the future with lots of cascading changes when set.